yt_dlp/extractor/twitter.py

```diff
@@ -1066,60 +1104,38 @@ class TwitterSpacesIE(TwitterBaseIE):
             'id': '1RDxlgyvNXzJL',
             'ext': 'm4a',
             'title': 'King Carlo e la mossa Kansas City per fare il Grande Centro',
-            'description': 'Twitter Space partecipated by annarita digiorgio, Signor Ernesto, Raffaello Colosimo, Simone M. Sepe',
+            'description': 'Twitter Space participated by annarita digiorgio, Signor Ernesto, Raffaello Colosimo, Simone M. Sepe',
             'uploader': r're:Lucio Di Gaetano.*?',
             'uploader_id': 'luciodigaetano',
             'live_status': 'was_live',
             'timestamp': 1659877956397,
         },
-        'params': {
-            'skip_download': True,  # requires ffmpeg
-        },
+        'params': {'skip_download': 'm3u8'},
     }, {
         # Twitter Space not started yet
         'url': 'https://twitter.com/i/spaces/1mnGeRyljQPJX',
         'info_dict': {
             'id': '1mnGeRyljQPJX',
             'title': '@staderlabs x Hedera AMA w/ @defigirlxoxo, @bmgentile, & @thehbarbull',
-            'description': 'Twitter Space partecipated by nobody yet.',
+            'description': 'Twitter Space participated by nobody yet',
             'uploader': 'Hedera',
             'uploader_id': 'hedera',
             'live_status': 'is_upcoming',
             'timestamp': 1662394668924,
         },
         'params': {
             'ignore_no_formats_error': True,
-            'skip_download': True,  # requires ffmpeg
+            'skip_download': True,
         },
         'expected_warnings': ['Twitter Space not started yet', 'No video formats found', 'Requested format is not available'],
     }]
```

Comment:
this test doesnt work

Reply:
I know. You have to put a twitter space url that is scheduled but not started yet.
When developing it that particular spaces was not broadcasted and it was useful to test the extractor error message. Now it's a normal downloadable Space.

It's a test that works only in certain situations.

Reply:
I'm pretty sure this isn't the expected result
```
❯ yt-dlp https://twitter.com/i/spaces/1mnGeRyljQPJX -v
[debug] Command-line config: ['--ignore-config', 'https://twitter.com/i/spaces/1mnGeRyljQPJX', '-v']
[debug] Encodings: locale cp65001, fs utf-8, pref cp65001, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.10.04 [4e0511f27] (source)
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Git HEAD: bbea6a65a
[debug] Python 3.10.8 (CPython 64bit) - Windows-10-10.0.22000-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg N-107787-gc469c3c3b1-20220814 (fdk,setts), ffprobe N-107787-gc469c3c3b1-20220814, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.1
[debug] Proxy map: {}
[debug] Loaded 1699 extractors
[debug] [twitter:spaces] Extracting URL: https://twitter.com/i/spaces/1mnGeRyljQPJX
[twitter:spaces] 1mnGeRyljQPJX: Downloading guest token
[twitter:spaces] 1mnGeRyljQPJX: Downloading JSON metadata
ERROR: 1mnGeRyljQPJX: An extractor error has occurred. (caused by KeyError('metadata')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "D:\Programs\Source\yt-dlp\yt-dlp\yt_dlp\extractor\common.py", line 674, in extract
    ie_result = self._real_extract(url)
  File "D:\Programs\Source\yt-dlp\yt-dlp\yt_dlp\extractor\twitter.py", line 1172, in _real_extract
    metadata = space_data['audioSpace']['metadata']
KeyError: 'metadata'
```

Reply:
I thought about creating a space scheduled 10 years in the future, but the limit is 30 days.

Reply:
This should probably use `traverse_obj` for `metadata = space_data['audioSpace']['metadata']` instead and check its result for the `Twitter Space not found` error?